elastic/kibana
# [Stack Monitoring] Migrate Index Views to React

Resolves https://github.com/elastic/kibana/issues/113631

- Implements Index Overview and Advanced View
- Implements "ItemTemplate" navigation, also in https://github.com/elastic/kibana/pull/113628, and may be removed depending which PR goes in first


### Test
- Navigate to Elasticsearch -> Indices -> click on an index
- Should land on the Overview index page
- Verify it refreshes data if auto refreshing is on
- Verify there is a shard legend with nodes by index.  The column should say "Nodes"
![Screen Shot 2021-10-01 at 2 19 08 PM](https://user-images.githubusercontent.com/1676003/135668409-ebc45914-658f-4e69-a12c-7e6816f8342f.png)
- Click on Advanced tab
- Verify it refreshes data if auto refreshing is on and looks the same in Angular mode
